docs/dev/design_documents/client_exit_codes.md

```diff
@@ -0,0 +1,63 @@
+# Chef-Client Exit Codes
+
+Signal outside tools of specific Chef-Client run status. Ability to determine results of a Chef-Client run.
```

Comment:
Do these two sentences describe Chef-Client Exit Codes? Does the second sentence describe something else?